addons/gamification/models/res_users.py

```diff
@@ -303,3 +336,20 @@ def get_gamification_redirection_data(self):
         """
         self.ensure_one()
         return []
+
+    def action_karma_report(self):
+        self.ensure_one()
+        return {
+            'name': _('Karma Tracking'),
+            'res_model': 'gamification.karma.tracking',
+            'target': 'current',
+            'type': 'ir.actions.act_window',
+            'view_mode': 'tree',
+            'context': {
+                'create': True,
```

Comment:
Do we need to re-specify create / delete / editable? They seem identical to view options.